goamiddleware/jwt_token_context.go

```diff
@@ -6,8 +6,8 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/fabric8-services/fabric8-cluster/token"
 	"github.com/fabric8-services/fabric8-common/log"
+	"github.com/fabric8-services/fabric8-common/token"
 
```

Comment:
I was expecting that this file will not be there at all in each_service .. rather each_service would be using the jwt_token_context_middleware from https://github.com/fabric8-services/fabric8-common/blob/master/goamiddleware/jwt_token_context.go

Reply:
this middleware in f8-cluster [link](https://github.com/fabric8-services/fabric8-cluster/blob/master/goamiddleware/jwt_token_context.go#L20) and f8-common [link](https://github.com/fabric8-services/fabric8-common/blob/master/goamiddleware/jwt_token_context.go#L22) are similar and meant for same purpose I guess.  In that case, we should remove this file from f8-cluster [link](https://github.com/fabric8-services/fabric8-cluster/blob/master/goamiddleware/jwt_token_context.go) and directly use middleware from f8-common.

Reply:
The current middleware in common was taken from WIT. It's outdated and has some issues. We should really move the middleware from cluster/auth to common and re-use it here.
With modifications suggested by Nurali - https://chat.openshift.io/developers/pl/awuewy3jubde78nw4q94oqf7gc

Reply:
See https://github.com/fabric8-services/fabric8-common/pull/27